src/main/java/com/sayone/ebazzar/common/RestResources.java

```diff
@@ -20,4 +20,23 @@
     String PRODUCT_POST_SAVE = "/create";
     String PRODUCT_PUT_UPDATE = "/update/{id}";
     String PRODUCT_GET_SEARCH = "/search";
+
+    /*
+     * API Endpoints for Review related operations
+     * */
+//    String REVIEW_ROOT = "/customer";
```

Comment:
why is it commented ? if it is not need, don't commit else give proper doc comments.